shared/provision/code-page/index.tsx

```diff
@@ -342,7 +343,13 @@ const Qr = (props: Props) =>
 const EnterText = (props: Props & {code: string; setCode: (code: string) => void}) => {
   const {code, setCode} = props
   const {onSubmitTextCode} = props
-  const onSubmit = React.useCallback(() => onSubmitTextCode(code), [code, onSubmitTextCode])
+  const onSubmit = React.useCallback(
+    e => {
+      e.preventDefault()
```

Comment:
this was confusing before. since its multiline pressing enter actually adds a newline, YET it also submits. this stops the newline from being added